payoneer-client.gemspec

```diff
@@ -14,10 +14,10 @@ Gem::Specification.new do |s|
 
   s.extra_rdoc_files = ['README.md']
 
-  s.required_ruby_version = '~> 2.0'
+  s.required_ruby_version = '~> 2.2'
```

Comment:
You can leave this at 2.0, but upgrade the gem version above, to 0.4

Reply:
Hmm it yells at me when I change it to 2.2 because it's not equal to TargetRubyVersion of 2.2 in rubocop.yml